release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2364,3 +2364,82 @@ link:https://access.redhat.com/solutions/6001181[{product-title} 4.7.9 container
+
+[id="ocp-4-7-11-installation-existing-iam-roles-aws"]
+===== Using existing IAM roles for clusters on AWS
+
+You can now define a pre-existing AWS identity access management (IAM) role for your machine instance profiles by setting the `compute.platform.aws.iamRole` and `controlPlane.platform.aws.iamRole` fields in the `install-config.yaml` file. This allows you to match naming schemes and include predefined permissions boundaries for your IAM roles for clusters installed on AWS.
+
+[id="ocp-4-7-11-installation-pre-existing-hosted-zones-aws"]
+===== Using pre-existing Route53 hosted private zones on AWS
+
+You can now define an existing Route 53 private hosted zone for your cluster by setting the `platform.aws.hostedZone` field in the `install-config.yaml` file. You can only use a pre-existing hosted zone when also supplying your own VPC.
+
+[id="ocp-4-7-11-old-oauth-warning-message"]
+===== Informational alert for new OAuth format
+
+In an upcoming release of {product-title}, OAuth tokens that do not include a SHA-256 prefix can no longer be used or created. With this release, an informational alert notifies the administrator of a cluster that contains OAuth tokens without the prefix about the planned behavior change. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1949941[*BZ#1949941*].
+
+[id="ocp-4-7-11-bug-fixes"]
+==== Bug fixes
+
+* Previously, when a user had permission to create a resource, but not permission to edit it, the web console YAML editor was incorrectly set to read-only mode. The editor content is now editable by users with create access for the resource.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1942027[*BZ#1942027*])
+
+* Previously, both the Cloud Credential Operator (CCO) and the Cluster Version Operator (CVO) reported if the CCO deployment was unhealthy. This resulted in double reporting if there was an issue. With this release, the CCO no longer reports if its deployment is unhealthy.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1948702[*BZ#1948702*])
+
+* Previously, namespaces were missing from the Machine Config Operator `relatedObjects` resource, and logs for some on-premise services were not collected in `must-gather` as a result. With this release, the required namespaces are added to the Machine Config Operator `relatedObjects` resource, and logs for on-premise services are collected in `must-gather`.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1950498[*BZ#1950498*])
+
+* Previously, the kubelet would sometimes overwhelm the API servers by opening a large number of watch requests for secrets and config maps, particularly on node reboot. With this release, the number of kubelet watch requests is reduced to reduce the load on API servers.
```

Comment:
Not critical to change, but "is reduced to reduce" seems repetitive, maybe there's a different term you could use for one of them?

Reply:
I'll make one a decrease - for some reason that word escaped me earlier :)